tests/test_main.py

```diff
@@ -0,0 +1,369 @@
+import pytest
+import datetime
+
+from django.contrib.auth import get_user_model
+from django.conf import settings
+from django.db.utils import IntegrityError
+from django.core.exceptions import ValidationError
+
+from db.models import (
+    Actor,
+    Genre,
+    Movie,
+    MovieSession,
+    CinemaHall,
+    Order,
+    Ticket
+)
+from services.movie_session import (
+    get_tickets_of_taken_seats
+)
+from services.user import create_user, get_user, update_user
+from services.order import create_order, get_orders
+
+
+@pytest.fixture()
+def database_data():
```

Comment:
This is not a best practice usage of pytest fixtures - this style is more `unittest` like.
In pytest you'd better split everything in more specific fixtures, and if needed - make fixture depend on another fixutre.
So, 1 fixture for genres, another for actors, and after that movie fixture, that depends on both :)
Of course same with other ones.
And in tests - make each test depend only on those fixture you need in this test :)

You will see, how your code become much more clean and pretty